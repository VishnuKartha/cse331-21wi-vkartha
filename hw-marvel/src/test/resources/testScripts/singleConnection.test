#Tries to find paths of a graph loaded from an tsv file with a single connection

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/singleConnection.tsv


LoadGraph singleConnection singleConnection.tsv

FindPath singleConnection VISHNU LEBRON_JAMES
FindPath singleConnection LEBRON_JAMES VISHNU
FindPath singleConnection LEBRON_JAMES KEVIN_DURANT
