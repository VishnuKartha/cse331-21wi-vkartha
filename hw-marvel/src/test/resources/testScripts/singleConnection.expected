#Tries to find paths of a graph loaded from an tsv file with a single connection

# To run this test, the marvel dataset must exist at
# hw-marvel/src/main/resources/data/singleConnection.tsv


loaded graph singleConnection

path from VISHNU to LEBRON JAMES:
VISHNU to LEBRON JAMES via basketballTeam
path from LEBRON JAMES to VISHNU:
LEBRON JAMES to VISHNU via basketballTeam
unknown character KEVIN DURANT
